src/main/java/fr/minuskube/inv/util/ItemBuilder.java

```diff
@@ -0,0 +1,275 @@
+     * @param amount The amount of the item that should be used
+     * @return A new ItemBuilder instance
+     */
+    public static ItemBuilder builder(Material material, int amount) {
+        return new ItemBuilder(material, amount);
+    }
+
+    /**
+     * Create a new ItemBuilder instance creating an ItemStack with the given material
+     *
+     * @param material The material to use for the ItemStack
+     * @return A new ItemBuilder instance
+     */
+    public static ItemBuilder builder(Material material) {
+        return builder(material, 1);
+    }
+
+
+    private ItemBuilder(ItemStack stack) {
+        this.stack = new ItemStack(stack);
+        meta = stack.getItemMeta();
+    }
+
+    private ItemBuilder(Material material, int amount) {
+        stack = new ItemStack(material, amount);
+        meta = stack.getItemMeta();
+    }
+
+
+    /**
+     * Sets the displayName of the item to the given input
+     *
+     * @param displayName The new display name
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder name(String displayName) {
+        meta.setDisplayName(displayName);
+        return this;
+    }
+
+
+    /**
+     * Sets the lore for the item based on the string array passed to it
+     *
+     * @param lore The new lore, where each element is a new line
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder lore(String[] lore) {
+        return lore(Arrays.asList(lore));
+    }
+
+    /**
+     * Sets the lore for the item based on the string array passed to it
+     *
+     * @param lore The new lore, where each element is a new line
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder lore(List<String> lore) {
+        meta.setLore(lore);
+        return this;
+    }
+
+    /**
+     * Sets the lore to the given string. If a newline character (<code>\n</code>) is found, the lore will be split into an additional line
+     *
+     * @param lore The new lore, each line separated by a <code>\n</code>
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder lore(String lore) {
+        return lore(newlinePattern.split(lore));
+    }
+
+    /**
+     * Appends the given line to the end of the lore. Newline characters (<code>\n</code>) will be ignored
+     *
+     * @param line The line of lore, that should be appended to the existing lines
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder appendLore(String line) {
+        List<String> lore = meta.getLore();
+        if (lore == null) lore = new ArrayList<>();
+        lore.add(line);
+        meta.setLore(lore);
+        return this;
+    }
+
+    /**
+     * Removes a line from the current lore using {@link List#remove(Object)}
+     *
+     * @param line The line of lore, that should be removed
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder removeLore(String line) {
```

Comment:
Removing an object from the list especially strings shouldn't be like that. It should an integer that's line number to remove. If i add "test" string to the lore and use removeLore("test"); it won't be removed because of the Java.